charts/spire/charts/spire-agent/README.md

```diff
@@ -87,7 +87,7 @@ A Helm chart to install the SPIRE agent.
 | `nodeAttestor.tpmDirect.pubHash.image.tag`             | Overrides the image tag                                                                                                                                                                           | `v1.8.7`                                                                         |
 | `workloadAttestors.unix.enabled`                       | Enables the Unix workload attestor                                                                                                                                                                | `false`                                                                          |
 | `workloadAttestors.k8s.enabled`                        | Enables the Kubernetes workload attestor                                                                                                                                                          | `true`                                                                           |
-| `workloadAttestors.k8s.verification.type`              | What kind of verification to do against kubelet. Valid options are [hostCert, apiServerCA, skip]                                                                                                  | `skip`                                                                           |
+| `workloadAttestors.k8s.verification.type`              | What kind of verification to do against kubelet. Valid options are [auto, hostCert, apiServerCA, skip]                                                                                            | `skip`                                                                           |
```

Comment:
```suggestion
| `workloadAttestors.k8s.verification.type`              | What kind of verification to do against kubelet. auto will first attempt to use the apiServerCA, and then fall back on hostCert. Valid options are [auto, hostCert, apiServerCA, skip]                                                                                            | `skip`                                                                           |
```

Reply:
Its the other way around. Will update though with that pattern. It looks good.

Reply:
Updated. Please re-review

Reply:
looks good.